HW9/IhorBohush/Weather_pyGame/main.py

```
@@ -0,0 +1,97 @@
+        if re.search(r"\W+$", text_input):
+            text_input = re.sub(r"\W+$", "", text_input)
+    return text_input
+
+
+running = True
+while running:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            running = False
+        if event.type == pygame.KEYDOWN:
+            if input_active:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)